sdks/python/apache_beam/io/gcp/gcsfilesystem.py

```diff
@@ -45,6 +45,10 @@ class GCSFileSystem(FileSystem):
   CHUNK_SIZE = gcsio.MAX_BATCH_OPERATION_SIZE  # Chuck size in batch operations
   GCS_PREFIX = 'gs://'
 
+  def __init__(self, pipeline_options):
+    super().__init__(pipeline_options)
```

Comment:
another possibility is to initialize the global credential here, like:

_ = auth.get_service_credentials(pipeline_options)